sources/api/apiserver/src/server/mod.rs

```diff
@@ -354,6 +363,43 @@ async fn get_configuration_files(
     Ok(ConfigurationFilesResponse(resp))
 }
 
+/// Get the update status from 'thar-be-updates'
+async fn get_update_status() -> Result<UpdateStatusResponse> {
+    let lockfile = File::create(UPDATE_LOCKFILE).context(error::UpdateLock)?;
+    lockfile.try_lock_shared().context(error::UpdateShareLock)?;
+    let result = thar_be_updates::status::get_update_status();
+    lockfile.unlock().context(error::Unlock)?;
+    match result {
+        Ok(update_status) => Ok(UpdateStatusResponse(update_status)),
+        Err(e) => match e {
+            thar_be_updates::error::Error::OpenStatusFile { .. } => {
```

Comment:
```suggestion
            thar_be_updates::error::Error::NoStatusFile { .. } => {
```
Or `Missing`... but this made me think something else had the status file open.